Tests/test_pile_ui.gd

```diff
@@ -0,0 +1,44 @@
+extends GutTest
+#test for ui pop up
+
+var _card_container_scene: PackedScene = load("res://Cards/CardContainer.tscn")
+var _card_container_scroll_scene: PackedScene  = load("res://#Scenes/CardScrollUI.tscn")
+var _card_container: CardContainer = null
+var _card_scroll: Control = null
+
+const SCROLLCONTAINER = 2
+const GRIDCONTAINER = 0
+
+
+func _test_setter():
```

Comment:
Instead of calling this `_test_setter()` on the `test_populate()` function, you can override `before_all()` which is an override in GutTest that will do initial setup of stuff before running each test! This'll basically just remove `_test_setter()` from `test_populate()` and automatically provide the setup for any other tests we may make in this file.

https://gut.readthedocs.io/en/latest/Creating-Tests.html